dashboard/app/mailers/pd/application/teacher_application_mailer.rb

```diff
@@ -22,37 +22,37 @@ def confirmation(teacher_application)
       end
     end
 
-    def principal_approval_teacher_reminder(teacher_application)
+    def administrator_approval_teacher_reminder(teacher_application)
```

Comment:
Sorry, remind me, this is part of a larger move to rename "principal" to "administrator" right?

Reply:
This is part of that but actually we are going to move these changes to another task so that all the renaming happens in one fell swoop so it's less confusing! I'll change that shortly